README.md

```diff
@@ -497,7 +497,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [João Barreiros](https://github.com/j-barreiros)
   - [Justin Braun](https://github.com/justin-braun)
   - [Jyoti KM](https://github.com/JyotiKM29)
-
+  - [JesusGerardoAguiar](https://github.com/JesusGerardoAguiar)
```

Comment:
Please add your name in line 486, after Jessica Teo.

Reply:
Hi!, I added another commit with the requested changes :)